pandas/core/ops.py

```diff
@@ -1379,3 +1454,12 @@ def wrapper(self, other):
         name = name[2:-2]
     wrapper.__name__ = name
     return wrapper
+
+
+sparse_array_special_funcs = dict(arith_method=_arith_method_SPARSE_ARRAY,
```

Comment:
are these new?

Reply:
This changes zero logic, just puts a dict here instead of passing these directly in sparse.array.